src/app/actions.ts

```
@@ -227,6 +228,39 @@ export async function updateHolidayRequestStatus(input: UpdateStatusInput) {
 
+    // Handle Slack status updates for approved requests
+    if (status === 'approved') {
+      try {
+        // Get the user details for the request
+        const requestUser = await db
+          .select()
+          .from(users)
+          .where(eq(users.id, updatedRequest.userId))
+          .limit(1);
+
+        if (requestUser[0] && requestUser[0].slackPresenceUpdate && requestUser[0].slackEmail) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)